### PR TITLE
Simplify test model creation - use simplesbml

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: tests/test_lint.py [hunk 1]
```diff
@@ -3,12 +3,11 @@
 from math import nan
 from unittest.mock import patch
 
-import libsbml
 import pandas as pd
 import pytest
 
 import petab
-from petab import (lint, sbml)  # noqa: E402
+from petab import lint  # noqa: E402
```
Comment (dilpath) [inline tests/test_lint.py:10]: Why does only this need `noqa`?

File: tests/test_lint.py [hunk 2]
```diff
@@ -211,35 +210,32 @@ def test_assert_no_leading_trailing_whitespace():
...
 def test_assert_model_parameters_in_condition_or_parameter_table():
-    document = libsbml.SBMLDocument(3, 1)
-    model = document.createModel()
-    model.setTimeUnits("second")
-    model.setExtentUnits("mole")
-    model.setSubstanceUnits('mole')
-    sbml.add_global_parameter(model, 'parameter1')
-    sbml.add_global_parameter(model, 'noiseParameter1_')
-    sbml.add_global_parameter(model, 'observableParameter1_')
+    import simplesbml
```
Comment (dilpath) [inline tests/test_lint.py:213]: Move to header? Currently imported multiple times.

Also imported within test cases in other files.

File: tests/test_lint.py [hunk 3]
```diff
@@ -419,29 +415,29 @@ def test_check_condition_df(minimal_sbml_model):
...
 
     # fix:
-    sbml_model.createSpecies().setId('s1')
-    lint.check_condition_df(condition_df, sbml_model)
+    ss_model.addSpecies("[s1]", 1.0)
+    lint.check_condition_df(condition_df, ss_model.model)
 
     # compartment missing in model
-    condition_df['c1'] = [4.0]
+    condition_df['c2'] = [4.0]
```
Comment (dilpath) [inline tests/test_lint.py:434]: Compartment name changed? Is it that the default compartment in a `simplesbml.SbmlModel` is already `'c1'`?

File: tests/test_parameter_mapping.py [hunk 4]
```diff
@@ -263,20 +262,20 @@ def test_all_override(condition_df_2_conditions,
...
-        add_global_parameter(sbml_model, 'dynamicParameter1')
-        add_global_parameter(sbml_model, 'observableParameter1_obs1')
-        add_global_parameter(sbml_model, 'observableParameter2_obs1')
-        add_global_parameter(sbml_model, 'observableParameter1_obs2')
+        import simplesbml
+        ss_model = simplesbml.SbmlModel()
+        ss_model.addParameter('dynamicParameter1', 0.0)
+        ss_model.addParameter('observableParameter1_obs1', 0.0)
+        ss_model.addParameter('observableParameter2_obs1', 0.0)
+        ss_model.addParameter('observableParameter1_obs2', 0.0)
```
Comment (dilpath) [inline tests/test_parameter_mapping.py:278]: These observable parameters were removed in `tests/test_observables.py::test_get_output_parameters`, but kept here?

**Review by dilpath — APPROVED** [review]
Fine to merge as is

File: tests/test_lint.py (hunk 1 above)
Comment (dweindl) [reply tests/test_lint.py:10]: It doesn't.

File: tests/test_lint.py (hunk 2 above)
Comment (dweindl) [reply tests/test_lint.py:213]: The idea was not importing bulky libsbml everywhere, that when running non-sbml only, they will complete slightly faster (e.g., convenient when using auto-rerun).

File: tests/test_parameter_mapping.py (hunk 4 above)
Comment (dilpath) [reply tests/test_parameter_mapping.py:278]: Nevermind, misread the other test

**Pull request merged**